pkg/registry/storage/rest/storage_storage.go

```diff
@@ -119,6 +119,15 @@ func (p RESTStorageProvider) v1beta1Storage(apiResourceConfigSource serverstorag
 	}
 	storage["csidrivers"] = csiDriverStorage.CSIDriver
 
+	// register csistoragecapacity if CSIStorageCapacity feature gate is enabled
+	if utilfeature.DefaultFeatureGate.Enabled(features.CSIStorageCapacity) {
```

Comment:
According to the comment [here](https://github.com/kubernetes/kubernetes/pull/99641/commits/43e1dd87e167233f096d39dd1249b492506b8dd8#r587487569) it seems like we shouldn't feature gate this?

Reply:
We intentionally did not add support for individual resource enablement/disablement to `--runtime-config`, but kept enablement scoped to the group/version level. Verify-scripts and CI processes in kube expect `--runtime-config=api/all=true` to enable all APIs, and this subverts that.

Reply:
So I simply remove the feature gate check here?

Reply:
For the sake of expedience I have done that in a separate commit. That way the change has an explicit commit comment explaining it and if we want the check after all, we can bring it back.